set.mm

```diff
@@ -5467,6 +5467,15 @@ use disjunction (although this is not required since definitions are
       ( wa simpl syl2anc ) ABGACDABHEFI $.
   $}
 
+
+  ${
+    sylibda.1 $e |- ( ph -> ( ps <-> ch ) ) $.
+    sylibda.2 $e |- ( ( ph /\ ch ) -> th ) $.
+    $( A syllogism deduction.  (Contributed by SN, 16-Jul-2024.) $)
+    sylibda $p |- ( ( ph /\ ps ) -> th ) $=
+      ( biimpa syldan ) ABCDABCEGFH $.
+  $}
+
```

Comment:
just as in #4220, sylibda should've been named sylbida...

Reply:
I will fix that, bit it seems to result in a merge conflict anyway...

Reply:
Done

Reply:
Oops I fixed that too. Don't know if there will be a conflict...

Reply:
Don't worry, if there are merge conflicts, they can be solved easily (in this case).